scripts/draft-blog-post.mjs

```diff
@@ -49,6 +49,7 @@ const categories = [
   { dir: "yaml", title: "YAML" },
   { dir: "api", title: "API" },
   { dir: "cli", title: "CLI" },
+  { dir: "misc", title: "MISC" },
```

Comment:
I guess we no longer need this change as well as the diff in `scripts/utils/changelog-categories.mjs`

Reply:
I was told to create a misc folder in order for other non code changes to the website like mine.

Reply:
IC. I guess that’s done in https://github.com/prettier/prettier/pull/12412/files#diff-f35f14f178b81f6f55c22d476409f83519ce5d78b53bdca17d53b147698adc3eR52 now